src/components/Comments/Comment.js

```diff
@@ -254,7 +255,7 @@ const Comment = ({
           onLongPress={handleOnLongPress}
           style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
           onPress={() => onBlockComponent(comment)}>
-          <IconEn name="block" size={15.02} color={COLORS.gray9} />
+          <IconEn name="block" size={15.02} color={colors.gray1} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the block icon has been changed. Ensure that this change aligns with the overall design and color scheme of the application.
```diff
- <IconEn name="block" size={15.02} color={COLORS.gray9} />
+ <IconEn name="block" size={15.02} color={colors.gray1} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->